src/popup/router/components/Header.vue

```diff
@@ -85,6 +85,12 @@ export default {
     display: flex;
     justify-content: center;
     align-items: center;
+    color: $white-1;
+
+    .title {
+      font-size: 18px;
+      font-weight: 300;
```

Comment:
This is overridden by `font-weight: 500;` from below, proposing to remove that line.

Reply:
It should be regular by design so I removed both to be default